packages/core-suggest/core-suggest.js

```diff
@@ -49,6 +84,25 @@ export default class CoreSuggest extends HTMLElement {
 
   escapeHTML (str) { return escapeHTML(str) }
 
+  _clearLiveRegion () {
```

Comment:
I suspect it may be useful to invoke this, and `_pushToLiveRegion` directly for users. Not entirely certain if the clear-function is as necessary/useful though.